waitForContainerSetup.sh

```diff
@@ -37,3 +40,4 @@ function waitUntilServiceIsReady() {
 waitUntilServiceIsReady gargoyleStsIsReady "Gargoyle STS"
 waitUntilServiceIsReady cephIsReady "Ceph"
 waitUntilServiceIsReady rangerAdminIsReady "Ranger Admin"
+waitUntilServiceIsReady gargoyleKeycloakIsReady "Keycloack ready"
```

Comment:
you only have to specify the name of the service without ready, so "Keycloak" is enough

Reply:
done